src/components/m-dropdown/index.js

```diff
@@ -44,8 +44,6 @@ class AXADropdown extends BaseComponentGlobal {
 
 wcdomready(() => {
   window.customElements.define(AXADropdown.tagName, AXADropdown);
-
-  BaseComponentGlobal.appendGlobalStyles(styles);
```

Comment:
mmmm it had a meaning to append it globally but i cant remeber why. The issue is, that  the BaseComponentGlobal should cache it and therefore the caching is not working anymore

Reply:
it's not working because the second parameter is misisng, i.d. the `nodeName`.

```js
// should include nodeName or tagName
BaseComponentGlobal.appendGlobalStyles(styles, AXADropdown.tagName);
```

Reply:
caching works fine

Reply:
The only diference I see is:
- By using `BaseComponentGlobal` the styles get only appended upon it's first usage/instance of `<axa-button>`
- By adding it manually it too the styles get append always upon `DOMReady`, nevertheless if the button was used/instantiated or not